content/upload/upload_1.md

```diff
@@ -0,0 +1,413 @@
++++
+title = "Data upload"
+weight = 1
+post = ""
++++
+
+{{% attachments style="blue" %}}
+{{% /attachments %}}
+
+{{< tabs groupId="select" >}}
+
+{{% tab name="Python" %}}
+
+One option is to load the sample data to the warehouse using Python as described in the below steps. Please download both the *sample_events.csv* and the *snowflake_upload.py* files from the attachments at the top of this page as you will need both.
+
+#### **Step 1:**  Set up your environment
+
+Set up a virtual environment (recommended) and install the [snowflake-connector-python](https://pypi.org/project/snowflake-connector-python/) package (tested with version 2.7.12).
+
+```python
+python3 -m venv env
+source env/bin/activate
+pip install snowflake-connector-python==2.7.12
+```
+
+
+#### **Step 2:** Change variables and connection details
+
+Open the snowflake_upload.py file and edit the following before you execute it:
+
+##### 2.1 Connection details - update username, password and account
+```python
+# Connection details - to be modified, where needed!
+conn=sf.connect(user='your_username',password='your_password',account='your_account')
+```
+
+##### 2.2 Variables to be modified - warehouse and database
+```python
+# Variables - to be modified, where needed!
+warehouse='YOUR_WAREHOUSE'
+database = 'YOUR_DB
+```
+##### 2.3 Path to the sample_data.csv
+```python
+# Path to be modified:
+csv_file = '/Users/your_user/path_to_csv/sample_events.csv'
+```
+
+#### **Step 3:** Execute snowflake_upload.py
+
+It should finish execution within a minute. You should be alerted as soon as each intermediary step finishes:
+
+```
+Schema created
+Staging table YOUR_DB.ATOMIC.SAMPLE_EVENTS_STAGED is created
+Stage dropped, if applicable
+Stage created
+File put to stage
+Data loaded into staging table
+Target table: YOUR_DB.ATOMIC.SAMPLE_EVENTS is created
+Staging table: YOUR_DB.ATOMIC.SAMPLE_EVENTS_STAGED is dropped
+```
+Now you should have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data.
```

Comment:
Change to
"You will now have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data."